mypy_boto3_builder/stubs_static/types-aioboto3/session.pyi

```diff
@@ -91,7 +91,7 @@ class ResourceCreatorContext:
     async def __aenter__(self) -> AIOBoto3ServiceResource: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aioboto3/blob/main/aioboto3-stubs/session.pyi